nixos/modules/services/x11/urxvtd.nix

```diff
@@ -13,8 +13,7 @@ in {
     default = false;
     example = true;
     description = ''
-      Enable urxvtd, the urxvt terminal daemon. To use urxvtd, first
-      "export RXVT_SOCKET=/run/user/$(id -u)/urxvtd-socket" then run
+      Enable urxvtd, the urxvt terminal daemon. To use urxvtd, run
       "urxvtc".
```

Comment:
@Mic92 Seems like a separate issue, then. Want to open one? I'll happily revise this documentation once that's available.